README.md

```diff
@@ -159,6 +124,43 @@ It is possible to run linters for certain paths only by using [micromatch](https
 }
 ```
 
+### Multiple glob patterns
+
+It is also possible to specify multiple file filter patterns using `includes` and `excludes`:
+
+**Simple format example:**
+
+```diff json
+{
+- "*": "my-task",
++ "my-task-name": {
++   "includes": ["*"],
++   "excludes": ["*.ignore.*"],
++   "commands": "my-task"
++ }
+}
+```
+
+**Advanced format example:**
+
+```diff json
+{
+  "linters": {
+    "*.css": "stylelint",
+-   "*.js": ["prettier --write", "git add"],
++    "make-up": {
```

Comment:
Nitpick, but I think "prettify" is a better fit for that task

Reply:
Ha ha.. sure. I wanted to be clear that the name is completely unrelated to the commands.

Btw.. I only updated the readme.. yet to update the source code.

Reply:
Thanks for the hard work on this! Appreciate that a lot.